pageObjects/cartPage/fragments/cart.list.ts

```diff
@@ -0,0 +1,73 @@
+import { ElementArray, Element } from '../../../types'
+import {normalizePrice} from '../../../helpers/helpers'
+import { CartPage } from '../cart.page';
+
+export class CartList {
+
+
+
+    private get items(): ElementArray{
+        return $$('[class*="cart-list__item"]')
+    }
+
+    private get counterInput(): Element {
+        return $('[data-testid="cart-counter-input"]');
+    }
+
+    private get productPrices(): ElementArray {
+        return $$('[data-testid="cost"]')
+    }
+
+    private get productTitle(): Element {
+        return $('[data-testid="title"]')
+    }
+
+    private get kebabMenu(): Element{
+        return $('#cartProductActions0')
+    }
+
+    private get deleteButton(): Element {
+        return this.kebabMenu.nextElement().$('button')
+    }
+
+    async kebabMenuClick(): Promise<CartPage> {
+        await this.kebabMenu.click()
+    
+        return new CartPage()
+    }
+    
+    async deleteButtonClick(): Promise<CartList> {
+        await this.deleteButton.click()
+    
+        return this
+    }
+
+    getCounterInputValue(): Promise<string> {
+        return this.counterInput.getValue()
+    }
+
+    async getProductPrice(number): Promise<number> {
```

Comment:
I would get product by name instead of index

Reply:
Done.